configure.scss

```diff
@@ -0,0 +1,53 @@
+$_n-ui-features: (
+	ads: false,
+	buttons: false,
+	grid: false,
+	colors: false,
+	icons: false,
+	typography: false,
+	cookieMessage: false,
```

Comment:
The reason I quite like sorting modules into folders is that there's no need to lookup and open files, it only takes a glance to see a directory structure - either locally or on Github.

I think the ability to quickly lookup what's bundled by default and what isn't is very useful but not wedded to any particular way of doing that.